pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -0,0 +1,104 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+//go:build linux_bpf
+// +build linux_bpf
+
+package http
+
+import (
+	"math/rand"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/DataDog/datadog-agent/pkg/network/java"
+	nettelemetry "github.com/DataDog/datadog-agent/pkg/network/telemetry"
+	"github.com/DataDog/datadog-agent/pkg/process/monitor"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	manager "github.com/DataDog/ebpf-manager"
+)
+
+var (
+	// path to our java USM agent TLS tracer
+	javaUSMAgentJarPath = ""
+	// randomID is used here as an identifier, simple proof of authenticity
+	// between the injected java process and the ebpf ioctl that receive the payload
+	randomID = int64(0)
+)
+
+type JavaTLSProgram struct {
+	processMonitor *monitor.ProcessMonitor
+	cleanupExec    func()
+}
+
+var _ subprogram = &JavaTLSProgram{}
+
+func newJavaTLSProgram(c *config.Config) *JavaTLSProgram {
+	if !c.EnableHTTPSMonitoring || !c.EnableJavaTLSSupport {
+		return nil
+	}
```

Comment:
I'd add a log telling it is disabled / enabled

Reply:
we don't log program that doesn't start due to a flag unset, like in ebpf_ssl, ebpf_gotls

Reply:
I added a log in Start() and Stop() if (p == nil) { log.Debug()}

Reply:
but added a log in Start() Stop() if the object is nil